### PR TITLE
Update several modules

### PR DESCRIPTION
**Pull request by oxpa**
Even though I precisely hit all the modules that are not compiled, I checked that no build is broken.

### CODE REVIEW

File: contrib/src/passenger/SHA512SUMS [hunk 1]
```diff
@@ -1 +1 @@
-b46864cd7fa83fe39a2a4f42d581660c9a521b96942604b56caa567adff272bb092e82a46a6817a7b68be4830ad1015c9af4429a6d5b26edfe9c3e3884046559  passenger-6.0.20.tar.gz
+5e01596bd658dad94b2281ad1efd5cb4624268021d4347cd911c14824e11e15125032b1793c2aa82e879cf07f93473fe78d9fc6a1f2b6f5fc0dcc12a84a5ea45 passenger-6.0.23.tar.gz
```
Comment (thresheek) [inline contrib/src/passenger/SHA512SUMS:1]: Hmmm, that seems suspicious - we had two spaces as a delimiter before, and now it's one.  What changed?

Comment (oxpa) [reply contrib/src/passenger/SHA512SUMS]: just an artifact of manual editing the file, wasn't careful enough. Put two spaces back

File: docs/nginx-module-headers-more.xml [hunk 2]
```diff
@@ -5,6 +5,17 @@
 <change_log title="nginx_module_headers_more">
 
 
+<changes apply="nginx-module-headers-more" ver="0.37" rev="1" basever="1.27.2"
+         date="2024-10-17" time="16:59:46 +0100"
+         packager="Nginx Packaging &lt;nginx-packaging@f5.com&gt;">
+<change>
+<para>
+Updated headers-more module to 0.35 release.
```
Comment (thresheek) [inline docs/nginx-module-headers-more.xml:13]: Should be 0.37 here.

**Comment by thresheek** [comment]
Otherwise, LGTM.

Thanks!

File: docs/nginx-module-headers-more.xml (hunk 2 above)
Comment (oxpa) [reply docs/nginx-module-headers-more.xml]: now this was a silly typo...

**Pull request merged**